notebooks/interfield_differences.ipynb

```diff
@@ -0,0 +1,1245 @@
+{
```

Comment:
<p>I assume this is the opposite of the previous cell (papers that did surprisingly well in AI and poorly in comp bio)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Yep!</p>